index.html

```diff
@@ -0,0 +1,282 @@
+                  fill="#DBD8D7"
+                />
+              </svg>
+            </a>
+            <a href="https://twiter">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="40"
+                height="40"
+                viewBox="0 0 40 40"
+                fill="none"
+              >
+                <path
+                  d="M29.7665 10.3698C29.4331 9.97109 28.8665 9.87143 28.4331 10.1704L26.4998 11.4993C25.5998 10.5359 24.3331 10.0043 22.9665 10.0043C20.2665 10.0043 18.0331 12.197 18.0331 14.9212C18.0331 15.3199 18.0665 15.6854 18.1665 16.0508C15.8331 15.5525 14.4331 13.8581 13.8998 10.8017C13.8331 10.403 13.5331 10.104 13.1331 10.0043C12.7665 9.90465 12.3665 10.0708 12.1331 10.3698C12.0665 10.4362 6.56646 18.0774 14.5331 24.5558C13.7665 26.1837 11.4665 27.579 10.5665 28.0441C10.1331 28.2435 9.93313 28.7086 10.0331 29.1737C10.1331 29.6388 10.5331 29.9378 10.9998 29.9378H12.7331C22.8665 29.9378 27.4665 24.9212 27.5665 13.692L29.6665 11.6987C30.0665 11.3664 30.0998 10.7684 29.7665 10.3698Z"
+                  fill="#DBD8D7"
+                />
+              </svg>
+            </a>
+            <a href="https://medium">
+              <svg width="40" height="40" viewBox="0 0 40 40" fill="none">
+                <path
+                  d="M28.4042 13.8804L30 12.3377V12H24.4717L20.5317 21.9099L16.0492 12H10.2525V12.3377L12.1167 14.6055C12.2983 14.7731 12.3933 15.0181 12.3692 15.2648V24.1768C12.4267 24.4977 12.3233 24.8278 12.1 25.0611L10 27.6328V27.9663H15.9542V27.6286L13.8542 25.0611C13.6267 24.8269 13.5192 24.5027 13.565 24.1768V16.4682L18.7917 27.9705H19.3992L23.8933 16.4682V25.6312C23.8933 25.8728 23.8933 25.9225 23.7367 26.0808L22.12 27.6615V28H29.9633V27.6623L28.405 26.1204C28.2683 26.016 28.1975 25.8408 28.2267 25.6707V14.3301C28.1975 14.1592 28.2675 13.984 28.4042 13.8804Z"
+                  fill="#DBD8D7"
+                />
+              </svg>
```

Comment:
[OPTIONAL] Your design is perfect. You took care of minor details. I know you will be refining and organizing your project in the next milestones. But don't you think, organizing your code from the very start will help you work easily in the future?
So the SVG background images and icons that you took from Figma, can be put in a separate folder named let's say `images` or `images/icons` as you feel convenient. And then importing icons or background images in the  HTML or CSS files. This will eliminate the clutter. Because you are going to write a lot of code in this project. HAPPY CODING !!!

Reply:
Thank you for the highlights, I will take note of that in subsequent works.